Snake/BusinessLayer/Engine.cs

```diff
@@ -116,8 +117,12 @@ public Engine(int length, int width, int mode)
 
 
                 case food:  // snake hits the food
+                    foreach (Point value in gameSnake1.returnCurrentSnakePosition())
```

Comment:
value is not really a good description of the variable - something more descriptive  - snakePoint ? snakeSegment? ???